### DISCUSSION TITLE
How to set the default theme when using theme-change

### DISCUSSION THREAD

**Discussion by sbahman** [saadeghi/daisyui#1574] [Q&A] [answered]
I have defined my own `custom` theme and am also enabling a couple of additional themes in my tailwind.config:

```
daisyui: {
    themes: [
      {
        custom: {
          primary: '#7e22ce',
          secondary: '#a21caf',
          accent: '#d97706',
          neutral: '#57534e',
          'base-100': '#312342',
          info: '#3ABFF8',
          success: '#36D399',
          warning: '#FBBD23',
          error: '#F87272',
        },
      },
      'cupcake',
      'dark',
      'synthwave',
      'valentine',
    ],
```
Even though I put the `custom` theme as the first theme in the list, the page always defaults to the `dark` theme for first time visitors. After changing the theme using `theme-change` everything works as expected though.

How can I set my `custom` theme as the default theme to load the page with?

**Comment by saadeghi** [comment, marked as answer]
By default your first theme (on the array) would be the default theme and the theme named `dark` would be the default for when OS is using dark mode (`prefers-color-scheme: dark`)  
There is a config to change that. See `darkTheme` config here: https://daisyui.com/docs/config/

So if you want to totally ignore the OS dark mode and always show 1 specific theme by default, set it like `darkTheme: "custom",`

https://play.tailwindcss.com/iUUThvW3H2?file=config
